staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types.go

```diff
@@ -34,6 +34,31 @@ type CustomResourceDefinitionSpec struct {
 	Validation *CustomResourceValidation
 	// Subresources describes the subresources for CustomResources
 	Subresources *CustomResourceSubresources
+
+	// AdditionalPrinterColumns are additional columns shown e.g. in kubectl next to the name and creation time.
+	AdditionalPrinterColumns []CustomResourceColumnDefinition
```

Comment:
It looks like I'm overruled here. So I'm guessing the always present field will be `name` and this will be defaulted to `createdAt`?

Reply:
fixed

Reply:
Nit: s/created-at/createdAt